doc/man/config.rst

```diff
@@ -0,0 +1,48 @@
+Config file support
+===================
+
+Config files are supported by most subcommands of ``pkgdev`` from any of three
+locations. Listed in order of increasing precedence these include the
+following:
+
+- system config -- /etc/pkgdev/pkgdev.conf
+- user config -- ~/.config/pkgdev/pkgdev.conf
+- custom config -- specified via the --config option
+
+Any settings from a config file with higher precedence will override matching
+settings from a config file with a lower precedence, e.g. user settings
+override system settings. Note that command line options override any matching
+config file setting.
+
+In terms of file structure, basic INI formatting is required and allows
+creating a default section (DEFAULT) for system-wide settings or repo-specific
+sections. The INI key-value pairs directly relate to the available
+long-options supported by the various prefixed by the subcommand name and their
+related values. See the following examples for config settings:
+
+- Run ``pkgcheck scan`` before committing and confirm creating commit with QA
```

Comment:
```suggestion
- Run ``pkgcheck scan`` before committing and permit creating commits with QA
```

Reply:
(confirm makes it sound like it asks me first)

Reply:
It does ask you "are you sure".

Reply:
"asks for confirmation (instead of aborting) when creating commits with QA errors"